includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -3920,9 +3922,11 @@ public function save_upe_appearance_ajax() {
 			$appearance = apply_filters( 'wcpay_upe_appearance', $appearance, $is_blocks_checkout );
 
 			$appearance_transient = $is_blocks_checkout ? self::WC_BLOCKS_UPE_APPEARANCE_TRANSIENT : self::UPE_APPEARANCE_TRANSIENT;
+			$appearance_theme_transient = $is_blocks_checkout ? self::WC_BLOCKS_UPE_APPEARANCE_THEME_TRANSIENT : self::UPE_APPEARANCE_THEME_TRANSIENT;
```

Comment:
PHP linter is complaining about the `=` not being aligned here.

Reply:
This should also be resolved now after a merge. 👍